src/app/features/components/main/main.scss

```
@@ -0,0 +1,355 @@
+      }
+    }
+  }
+
+  @include media-tablet {
+    padding: 4rem 0;
+  }
+}
+
+.advantages {
+  background-color: var(--mat-sys-surface-container-low);
+  padding: 8rem 0;
+
+  &__title {
+    text-align: center;
+  }
+
+  &__container {
+    display: flex;
+    justify-content: space-around;
+    padding: 6rem 0 0;
+    flex-wrap: wrap;
+    row-gap: 1.5rem;
+  }
+
+  &__card {
+    padding: 1.5rem;
+    display: flex;
+    flex-direction: column;
+    row-gap: 2rem;
+    text-align: center;
+    width: 300px;
+    background-color: var(--mat-sys-surface);
+    border-color: var(
+      --mat-card-elevated-container-color,
+      var(--mat-sys-surface-container-low)
+    );
+    border-radius: var(
+      --mat-card-elevated-container-shape,
+      var(--mat-sys-corner-medium)
+    );
+    box-shadow: var(
+      --mat-card-elevated-container-elevation,
+      var(--mat-sys-level1)
+    );
+    @extend %transition;
+
+    &:hover {
+      box-shadow: 0 3px 7px grey;
+    }
+
+    &__image {
+      width: 100px;
+      height: 100px;
+      object-fit: cover;
+      background-repeat: no-repeat;
+      align-self: center;
+    }
+
+    &__title {
+      font-size: 2rem;
+      font-weight: 500;
+    }
+
+    &__text {
+      font-size: 1.6rem;
+      font-weight: 400;
+      line-height: 1.5;
+    }
+  }
+
+  @include media-tablet {
+    padding: 4rem 0;
+  }
+}
+.user-friendly {
+  background-image: url('../../../../assets/images/frendly_interface.png');
+}
+.tools {
+  background-image: url('../../../../assets/images/tools.png');
+}
+.workflow {
+  background-image: url('../../../../assets/images/workflow.png');
+}
+
+.owners {
+  padding: 8rem 0;
+
+  &__title {
+    text-align: center;
+  }
+
+  &__container {
+    display: flex;
+    justify-content: space-around;
+    flex-wrap: wrap;
+    row-gap: 1.5rem;
+    padding: 6rem 0 0;
+  }
+
+  @include media-tablet {
+    padding: 4rem 0;
+  }
+}
+
+.rs-link {
+  width: 60%;
+  border: 2px solid var(--mat-sys-primary);
+  border-radius: 10px;
+  margin: 0 auto 1rem;
+  padding: 10px;
+  @extend %transition;
+
+  @include media-tablet {
+    width: 90%;
+  }
+
+  @include media-mobile {
+    width: 100%;
+  }
+
+  a {
+    text-decoration: none;
+    color: var(--mat-sys-primary);
+    padding: 1.5rem;
+
+    &:hover {
+      color: var(--mat-sys-on-surface);
+    }
+  }
+
+  &:hover {
+    box-shadow: 0 3px 7px grey;
+    background-color: var(--mat-sys-surface-container);
+  }
+}
+
+.rs-information {
+  display: flex;
+  justify-content: center;
+  column-gap: 2rem;
+  align-items: center;
+
+  @include media-tablet {
+    column-gap: 0.5rem;
+  }
+
+  .logo-rs {
+    width: 5rem;
+    height: 5rem;
+    background-image: url('../../../../assets/images/rss-logo.svg');
+    background-repeat: no-repeat;
+
+    &__message {
+      font-size: 1.8rem;
+      font-weight: 400;
+      display: flex;
+      flex-direction: column;
+      row-gap: 1.5rem;
+      font-weight: 600;
```

Please remove the repeated code

```suggestion
    &__message {
      font-size: 1.8rem;
      font-weight: 400;
      display: flex;
      flex-direction: column;
      row-gap: 1.5rem;
```

---

done